libs/dataset_deployer.py

```diff
@@ -4,37 +4,20 @@
 import csv
 import io
 import logging
-import boto3
 
 _logger = logging.getLogger(__name__)
 
 
 class DatasetDeployer(object):
-    """Common deploy operations for persisting files to s3 or local folder.
+    """Common deploy operations for persisting files to a local folder.
```

Comment:
I think there might be a couple other references to s3 somewhere (maybe in some of the cli commands).  can you look for those and clean them up?